datahub-web-react/src/app/ingest/source/builder/RecipeForm/constants.ts

```diff
@@ -74,7 +75,18 @@ import {
     TOPIC_DENY,
 } from './kafka';
 
-export const RECIPE_FIELDS = {
+interface RecipeFields {
```

Comment:
out of curiosity, why did you opt for `interface` here? I usually use type

Reply:
so as far as I understand, the two are very often interchangeable. I usually default to `interface` because `interface` names show up in error messages which helps for debugging and have some extra functionality like extending (not that I'm using that here). I have always used `type` for basically giving a name to a primitive type for readability.

I don't think either are wrong and neither is the obvious right choice either!